__tests__/app.test.js

```diff
@@ -177,3 +178,71 @@ describe("/api/articles", () => {
     });
 });
 
+describe("GET /api/articles/:article_id/comments", () => {
+    test("GET: 200 - responds with an array of comments for the given article_id", () => {
+        return request(app)
+            .get("/api/articles/1/comments")
+            .expect(200)
+            .then(({ body: { comments } }) => {
+                expect(comments).toBeInstanceOf(Array);
+                if(comments.length === 0){
+                    return;
+                }
```

Comment:
It's not conventional to return from a test. Instead, let's write an assertion. What do you expect the length of the comments to be? You can look at the test data to find this out.

Reply:
A failed test will implicitly return and the following assertions in that test won't run . 👍